src/olympia/abuse/tests/assets/cinder_webhook.json

```diff
@@ -1,38 +1,6 @@
 {
     "event": "decision.created",
     "payload": {
-        "appeal": {
```

Comment:
This only makes sense in appeals, and this json is used to test "original" decisions.